.github/workflows/ci.yml

```diff
@@ -1,12 +1,24 @@
-name: CI 
+name: CI
 on:
   - push
   - pull_request
 
-jobs: 
+jobs:
   test:
     runs-on: 'windows-2019'
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
       - run: go test -gcflags=all=-d=checkptr -v ./...
+
+  build:
+    runs-on: 'windows-2019'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15.0'
```

Comment:
we should update the go version in the modules too, but I'm not sure if we want to do that before we make a 0.4.x release to address the breaking changes in moby https://github.com/moby/moby/pull/42307